ras_backstage/__init__.py

```diff
@@ -22,33 +22,36 @@
 
 collection_exercise_api = Namespace('collection-exercise',
                                     path='/backstage-api/v1/collection-exercise')
+collection_instrument_api = Namespace('survey', path='/backstage-api/v1/collection-instrument')
```

Comment:
This isn't originally from this PR but this Namespace should be called `'collection-instrument'` rather than `'survey'`